gpii/node_modules/flatMatchMaker/src/FlatMatchMaker.js

```diff
@@ -99,7 +99,10 @@
             if (solrec.priority !== undefined) {
                 accepted = true;
             }
-            solrec.disposition = accepted ? "accept" : "reject";
+            fluid.extend(solrec, {
+                disposition: accepted ? "accept" : "reject",
+                active: accepted ? true : false // always true if accepted
```

Comment:
Is this really right? Or rather, is it permanently right? Didn't we decide that someday we might want to configure a solution without activating it? Can we add a link/comment to any discussion and plans on the meaning of this ...

Reply:
well, for now it's right :) ... The canopy matchmaker will do that (configure a solution without activating it, depending on whether multiple solutions are of the same type, and what */enabled settings a user have in their NP set), but I've mostly left the flat MM alone as the "activate anything that matches anything in the users NP set" it's always been. I improve it if we think it makes sense. Alternatively, I can add more info to the function comments, like:

"No priorities will be taken into account and any matching solution will both be configured and started. There are no current plans to improve the flat matchmaker, rather efforts of making a more intelligent matchmaker is done in the canopy matchmaker."

if that is true, ofc

Reply:
btw, most of the functionality of handling priorities, conflicts, etc., is in the matchmaker utilities, so it wouldn't be huge amounts of work to add support for that in the flat matchmaker. the biggest work would probably lie in adding tests for it to ensure it does what's expected in the various scenarios we might throw at it

Reply:
I think just a comment like "We consider that any accepted solution will be activated because of the simple-minded policy adopted by the flat matchmaker" will work

Reply:
No it's fine, we've got no particular desire to improve the flat matchmaker, it's just that we need to understand what we are looking at : P

Reply:
ok, added the comment to the function doc